docs/gensxg.md

```diff
@@ -0,0 +1,30 @@
+# gensxg
+A CLI SXG generator.
```

Comment:
"A command-line tool for generating SXG (Signed HTTP Exchanges)."

CLI is not as common as, say, GUI (or even CUI). It's also ambiguous to some kind of people, e.g. it can also stand for Common Language Interface (typically known as Microsoft .NET).